cmd/config.go

```diff
@@ -24,6 +24,7 @@ const (
 	configFile              = "Furyfile"
 	httpsRepoPrefix         = "git::https://github.com/sighupio/fury-kubernetes"
 	sshRepoPrefix           = "git@github.com:sighupio/fury-kubernetes"
+	registryRepoPrefix      = "git::https://github.com/terraform-aws-modules"
```

Comment:
Hi @lzecca78 , why only terraform-aws-modules? 

Should we add another parameter on furyfile like cloud with values like: "aws" "azure" "gcp" to search on respective GitHub organizations? And perhaps a hardcoded struct that contains which Github organizations are "official" for us.

Nice add btw :blush:

Reply:
pretty simple @nutellinoit : this pr comes from the issue you wrote here :https://github.com/sighupio/fury-kubernetes-aws/issues/22 . That was related to the only terraform-aws-modules. Once it will be accepted, implement it into something that can be  customized for each cloud provider, is really simple btw. About the idea of many github organizations, imho at this time is overengineering. The xp approach that provides a really small step by step iteration flow, i believe is the right now in this situations. Once is ok, i will extend it pretty easily!

Reply:
i close this because it refers to outdated code :)